IBMigrationTool/InterfaceBuilderFileMigrator.swift

```diff
@@ -12,11 +12,13 @@ import Foundation
 /// when upgrading to `ThunderBasics` v2.0.0
 final class InterfaceBuilderFileMigrator {
     
+    // Don't migrate colours for now as `CGColor` isn't creatable from user defined attributes so
+    // this silently breaks a LOT of UI!
```

Comment:
Might be clearer to say "otherwise this would silently break...", sounds a little like this change is breaking shizz!